mcs/class/System/System.Net.NetworkInformation/LinuxNetworkInterface.cs

```diff
@@ -260,6 +261,16 @@ sealed class LinuxNetworkInterface : UnixNetworkInterface
 		bool android_use_java_api;
 #endif
 
+#if UNITY
+		const string LibLog = "/system/lib/liblog.so";
+		const string LibLog64 = "/system/lib64/liblog.so";
+
+		internal static bool IsRunningOnAndroid = File.Exists (LibLog) || File.Exists (LibLog64);
```

Comment:
Is this the best check? Did it come from some documentation or example?

Reply:
Stole it from here: https://github.com/Unity-Technologies/mono/blob/unity-main/mcs/class/corlib/System.IO/LogcatTextWriter.cs#L92

Reply:
I'll clean this up the duplicate code is pretty ugly